torch/testing/_internal/common_methods_invocations.py

```diff
@@ -858,6 +858,8 @@ def method_tests():
         ('scatter', (), (0, torch.tensor(0, dtype=torch.int64), 2.5), 'scalar_all_dim0', (), [0]),
         ('scatter_add', (M, S), (0, gather_variable((S, S), 1, M), (S, S)), 'dim0', (), [0]),
         ('scatter_add', (M, S), (1, gather_variable((M, S // 2), 0, S), (M, S // 2)), 'dim1', (), [0]),
+        ('scatter_add', (M, S), (0, gather_variable((S, S), 1, M), (M, S)), 'dim0_src_size_ge_index', (), [0]),
+        ('scatter_add', (M, S), (1, gather_variable((M, S // 2), 0, S), (M, S)), 'dim1_src_size_ge_index', (), [0]),
```

Comment:
@albanD , could you please confirm that the gradient wrt src is indeed tested here?

Reply:
Checking the `create_input` function in this file. It looks like when the size is given in a typle, it creates a gradient that requires grad.
And the gradcheck will check all inputs that require gradients. So it should be fine yes.